Application Lifecycle Management/api-boxes.md

```diff
@@ -1174,18 +2240,16 @@ Reconfigures the stack based on the changes. See example update a CloudFormation
 
 Creating a box of the CloudFormation service type.
 
-**Request**
```

Comment:
Please, consider mark line 2188 only with two  #
## Create and Launch a CloudFormation Box

In the same manner, mark all subtitles with # and reference them in the header of this page "CloudFormation Box" as an index table
Example: Create a CloudFormation box with template
Example: Modify the CloudFormation Template
Example: Launch a CloudFormation Box
Example: Update a CloudFormation Stack in Real-Time